SwiftUIHW/SwiftUIHW/CustomViews/CustomRectangle.swift

```diff
@@ -0,0 +1,85 @@
+// CustomRectangle.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Кастомная вью Логина
+struct CustomRectangle: View {
```

Comment:
```suggestion
struct CustomRectangleView: View {
```
Если это вью где в конце View?